src/server/pkg/pachrpc/pachrpc.go

```diff
@@ -0,0 +1,58 @@
+package pachrpc
+
+import (
+	"fmt"
+	"sync"
+
+	"golang.org/x/net/context"
+
+	"github.com/pachyderm/pachyderm/src/client"
+)
+
+var (
+	// pachClient is the "template" client that contains the original GRPC
+	// connection and that other clients returned by this library are based on. It
+	// has no ctx and therefore no auth credentials or cancellation
+	pachClient *client.APIClient
+
+	// pachClientMu guards pachClient
```

Comment:
This looks like it should be changed to a comment for the Once object rather than what I'm guessing was previously a mutex.